draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2729,6 +2783,10 @@ Internet-Draft                 ACTN POI                       July 2023
       from the topology model defined in [CLIENT-TOPO]: this gap has
       been identified in section 5.2.1;
```

Comment:
Not clear which gap we are referring to? This is not the correct section as 5.2.1 is about Multi-layer LAG setup...Or do you refer to the fact that LAG is not supported in CLIENT-TOPO? Not clear to me...

Reply:
it is a LAG configuration attribute which is missing: rephrased to improve clarity